editoast/src/schema/rolling_stock/light_rolling_stock.rs

```diff
@@ -61,7 +60,7 @@ pub struct LightRollingStock {
     pub metadata: DieselJson<RollingStockMetadata>,
     #[diesel(sql_type = Nullable<Jsonb>)]
```

Comment:
This needs to be updated

Reply:
sorry, I didn't understand what needs updating?

Reply:
You changed the type of the field so you should also update the type that diesel is expected to read.

Reply:
Nope my bad, this is fine, sorry 😖